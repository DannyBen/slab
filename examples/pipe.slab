| Some text.
p
p
  |
p
  | Some text.
p
  | Some text.
  | Some other text.
p
  | Some text.
  a Some link.
  | Some other text.
