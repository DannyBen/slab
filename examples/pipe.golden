Some text.
<p>
</p>
<p>
</p>
<p>
    Some text.
</p>
<p>
    Some text.
    Some other text.
</p>
<p>
    Some text.
    <a>
        Some link.
    </a>
    Some other text.
</p>
